doc/sphinx/user/run-aspect/parameters-overview/muparser-format.md

```diff
@@ -9,9 +9,9 @@ C++ in the code of the plugin, or a plugin that allows you to describe this
 formula in a symbolic way in the input file (see
 {ref}`parameters:Initial_20temperature_20model`). An example
 of this latter case is this snippet of code discussed in
-{ref}`5.2.2`:
+{ref}`sec:cookbooks:convection_box_3d`:
 
-``` prmfile
+```{literalinclude} ../../../../manual//cookbooks/convection_box_3d/doc/initial.part.prm
```

Comment:
```suggestion
```{literalinclude} ../../manual/cookbooks/convection_box_3d/doc/initial.part.prm
```

Reply:
I'm confused about these changes, they work as is?

Reply:
But they are prettier if you use the symlink :smile:. Let's be consistent about the way we reference the cookbooks and benchmarks directories, and since we specifically created the symlinks to reference them from the documentation directory, let's be consistent with that.